app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/AppGridView.kt

```diff
@@ -169,7 +170,7 @@ private fun RealAppsGridBundlePreview() {
 private fun AppsRowViewPreview() {
   AptoideTheme {
     AppsRowView(
-      appsList = listOf(randomApp, randomApp, randomApp, randomApp),
+      appsList = previewAppsListIdleState.apps,
```

Comment:
What's the difference here?

Reply:
Not really much difference, its just because that val already exists and can be reused